ci/test/ray_coverage.py

```diff
@@ -0,0 +1,69 @@
+import click
+import logging
+import os
+import subprocess
+import sys
+import tempfile
+
+COVERAGE_FILE_NAME = "ray_release.cov"
+
+
+@click.command()
+@click.argument("test_target", required=True, type=str)
+def main(test_target: str) -> None:
+    """
+    This script collects dynamic coverage data for the test target, and upload the
+    results to database (S3).
```

Comment:
We're not uploading anything to s3 now, right?

Reply:
preemptive comments ;)